shuup/core/models/_orders.py

```diff
@@ -824,6 +826,13 @@ def get_product_summary(self):
             .values_list("product_id", "quantity"))
         for product_id, quantity in lines:
             products[product_id]['ordered'] += quantity
+
+        from ._products import ShippingMode
```

Comment:
It can't be imported at the top of _orders since it causes circular import. Probably shouldn't but couldn't figure the reason fast enough.